ReSharper.FSharp/test/data/features/daemon/typeHints/Signatures - Locals 01.fs.gold

```diff
@@ -130,6 +130,21 @@ let (|String|) (x: MyStruct) : string = String(x.myString)
 
 let f1 (x & Int(i) & String(s)) = ()
 
+
+let _ = function x -> ()
```

Comment:
What do you think about splitting this test files into smaller ones, so it could be easier to think about unrelated cases separately?